CHANGELOG.rst

```diff
@@ -5,7 +5,7 @@ All notable changes to this project are documented in this file.
 
 [0.8.3-dev] in progress
 -----------------------
--
+- Add ``MempoolCheckLoop`` and ``RemoveTransaction`` to NodeLeader.py to resolve `#703 <https://github.com/CityOfZion/neo-python/issues/703>`_
```

Comment:
Can we make this message a "what was solved" instead of a "how" it was solved? i.e.
> Fix confirmed TX not being purged from mempool 

That way somebody who reads the changelog doesn't have to click the issue link to know what was resolved. If you add the link to this PR, then they could lookup "how" it is resolved if they're interested.